### ISSUE TITLE
Dev:  Name is not centered in Share Code page

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#19241]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to Expensify
2. Go to Settings > Share Code

## Expected Result: 
Name is centered
## Actual Result : 
Name isn't centered

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** main branch
**Reproducible in staging?:** needs reproduction
**Reproducible in production?:** needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
![image (1)](https://github.com/Expensify/App/assets/43996225/e954867d-51b6-4984-9b79-bb5d9e66edec)

**Expensify/Expensify Issue URL:**
**Issue reported by:** @aman-atg 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1684255556393729

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by hungvu193** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Dev: Name is not centered in Share Code page
### What is the root cause of that problem?
The title of the QRCode is not center because we didn't add the align property for the title here:
https://github.com/Expensify/App/blob/9111b90468cd92867b93bf823e37060af819686a/src/components/QRShare/index.js#L71-L75
### What changes do you think we should make in order to solve the problem?
Add `textAlign: center` style to this text will fix the propblem.

### What alternative solutions did you explore? (Optional)

**Comment by dukenv0307** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The name is not centered in Share Code page
### What is the root cause of that problem?
https://github.com/Expensify/App/blob/9111b90468cd92867b93bf823e37060af819686a/src/components/QRShare/index.js#L71-L89

In the title and subtitle, we don't set text-align: center
### What changes do you think we should make in order to solve the problem?
We should add textAlign="center" in both title and subtitle

### What alternative solutions did you explore? (Optional)

**Comment by stephanieelliott** [comment]
Hey @Santhosh-Sellavel looks like we already have 2 proposals on this one!

**Comment by Santhosh-Sellavel** [comment]
Yeah, straightforward one, both work. I would like to with @hungvu193 here. Let me know your thoughts! @francoisl

**Comment by francoisl** [comment]
Per our [styling guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/STYLING.md#inline-styles), we shouldn't use inline styles, so yes let's go with @hungvu193's proposal that will move the style to `styles.js`.

**Comment by hungvu193** [comment]
@francoisl @Santhosh-Sellavel https://github.com/Expensify/App/pull/19566 is now ready for review!

**Comment by Santhosh-Sellavel** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [x]  [@Santhosh-Sellavel] The PR that introduced the bug has been identified. Link to the PR: https://github.com/Expensify/App/pull/18636
> * [x]  [@Santhosh-Sellavel] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: https://github.com/Expensify/App/pull/8742#discussion_r1215147965
> * [ ]  [@Santhosh-Sellavel] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
> * [x]  [@Santhosh-Sellavel] Determine if we should create a regression test for this bug.
> * [x]  [@Santhosh-Sellavel] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
> * [ ]  [@MitchExpensify] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by stephanieelliott** [comment]
@Santhosh-Sellavel and @hungvu193 have been paid! 
@aman-atg I've extended an offer to you for the reporting bonus, please accept when you get a chance!

**Comment by stephanieelliott** [comment]
Oops, you're right @hungvu193! Issued speed bonus to both C and C+, and paid @aman-atg for reporting. Thanks!
